plugin/web/extensions/sidebar/core/components/content/Share/hooks.ts

```diff
@@ -135,15 +135,19 @@ function generatePrintView(payload?: string) {
   };
   const localizedDate = currentDate.toLocaleDateString("ja", options);
 
+  const themeColor = getComputedStyle(document.documentElement).getPropertyValue(
```

Comment:
Can you use getComputedStyle inside a sandboxed iframe????

Reply:
I think so. It does not require parent's window, right?

Reply:
Oh my mistake I think